integration-tests/src/test/java/org/apache/fineract/integrationtests/LoanOriginationValidationTest.java

```diff
@@ -172,6 +174,80 @@ public void uc2() {
+                    operationDate, 100.0, 5).numberOfRepayments(6)//
+                    .clientId(null) //
+                    .loanTermFrequency(6)//
+                    .loanTermFrequencyType(2)//
+                    .transactionProcessingStrategyCode(LoanProductTestBuilder.ADVANCED_PAYMENT_ALLOCATION_STRATEGY)//
+                    .repaymentEvery(1)//
+                    .repaymentFrequencyType(2)//
+            ;//
+            callFailedRuntimeException = Assertions.assertThrows(CallFailedRuntimeException.class,
+                    () -> loanTransactionHelper.applyLoan(applicationRequest02));
+            assertEquals(400, callFailedRuntimeException.getResponse().code());
+            Assertions.assertTrue(callFailedRuntimeException.getMessage().contains("The parameter `clientId` is mandatory."));
+
+            // Submitted Date null
+            final PostLoansRequest applicationRequest03 = applyLoanRequest(client.getClientId(), loanProductResponse.getResourceId(),
+                    operationDate, 100.0, 5).numberOfRepayments(6)//
+                    .submittedOnDate(null) //
+                    .loanTermFrequency(6)//
+                    .loanTermFrequencyType(2)//
+                    .transactionProcessingStrategyCode(LoanProductTestBuilder.ADVANCED_PAYMENT_ALLOCATION_STRATEGY)//
+                    .repaymentEvery(1)//
+                    .repaymentFrequencyType(2)//
+            ;//
+            callFailedRuntimeException = Assertions.assertThrows(CallFailedRuntimeException.class,
+                    () -> loanTransactionHelper.applyLoan(applicationRequest03));
+            assertEquals(400, callFailedRuntimeException.getResponse().code());
+            Assertions.assertTrue(callFailedRuntimeException.getMessage().contains("The parameter `submittedOnDate` is mandatory."));
+
+            // expected disbursement Date null
+            final PostLoansRequest applicationRequest04 = applyLoanRequest(client.getClientId(), loanProductResponse.getResourceId(),
+                    operationDate, 100.0, 5).numberOfRepayments(6)//
+                    .expectedDisbursementDate(null) //
+                    .loanTermFrequency(6)//
+                    .loanTermFrequencyType(2)//
+                    .transactionProcessingStrategyCode(LoanProductTestBuilder.ADVANCED_PAYMENT_ALLOCATION_STRATEGY)//
+                    .repaymentEvery(1)//
+                    .repaymentFrequencyType(2)//
+            ;//
+            callFailedRuntimeException = Assertions.assertThrows(CallFailedRuntimeException.class,
+                    () -> loanTransactionHelper.applyLoan(applicationRequest04));
+            assertEquals(400, callFailedRuntimeException.getResponse().code());
+            Assertions
+                    .assertTrue(callFailedRuntimeException.getMessage().contains("The parameter `expectedDisbursementDate` is mandatory."));
+        });
```

Comment:
@alberto-art3ch Please add test case that covers if loan product id is null, we got "The parameter `loanProductId` is mandatory."